programs/gateway/src/lib.rs

```diff
@@ -96,6 +96,20 @@ impl CallableInstruction {
     }
 }
 
+/// Struct containing revert options
+/// # Arguments
+/// * `revert_address` Address to receive revert.
+/// * `call_on_revert` Flag if on_revert hook should be called.
+/// * `revert_message` Arbitrary data sent back in on_revert.
+/// * `on_revert_gas_limit` Gas limit for revert tx.
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, Debug, PartialEq)]
+pub struct RevertOptions {
```

Comment:
Let's add `abort_address` in the structure as well (evm address), so the struct should no longer change in the future